glustercli/cmd/utils.go

```diff
@@ -57,6 +58,26 @@ func sizeToMb(value string) (uint64, error) {
 	return size, nil
 }
 
+func logn(n, b float64) float64 {
+	return math.Log(n) / math.Log(b)
+}
+
+func HumanReadable(value uint64) string {
```

Comment:
can you write some comments for this function , with Example input and output

Reply:
Done

Reply:
While you can retain the comment, there's no need to export this function as it isn't used outside of the CLI package.